lib/voorhees/json_api.ex

```diff
@@ -19,6 +19,33 @@ defmodule Voorhees.JSONApi do
     actual
   end
 
+  def assert_schema_contains(%{"data" => resource} = actual, expected) when is_map(resource) do
+    assert_schema_contains(put_in(actual["data"], [resource]), expected)
+  end
+  def assert_schema_contains(%{"data" => resources} = actual, expected) when is_list(resources) do
+    actual = Enum.map(resources ++ actual["included"] || [], fn(resource) -> _stringify_keys(resource) end)
+    _assert_schema_contains(actual, _stringify_keys(expected))
+  end
+
+  defp _assert_schema_contains(actual, expected) do
+    actual_types = Enum.map(actual, fn(resource) -> resource["type"] end)
+    expected_types = Map.keys(expected)
+
+    assert length(expected_types -- actual_types) == 0,
+      "Expected types: #{Enum.join(expected_types, ", ")}\nGot: #{Enum.join(actual_types, ", ")}"
+
+    Enum.each expected_types, fn(expected_type) ->
+      expected_attributes = _stringify_items(expected[expected_type]["attributes"])
```

Comment:
I'm hesitant to overdue it with pipe operator. I like the suggestion but it does feel like hammering nails